### PR TITLE
feat: CRN-847 Add external users to event speakers in the CMS

### PR DESCRIPTION
**Pull request by peterstarling**

### CODE REVIEW

**Comment by niezgoda** [comment]
Also CR the script in events schema: https://cloud.squidex.io/app/1613/schemas/events?tab=scripts

File: apps/crn-server/src/entities/event.ts [hunk 1]
```diff
@@ -45,28 +72,50 @@ export const parseEventSpeaker = (
...
 
+    if (user?.__typename === 'ExternalAuthors') {
+      return {
+        externalUser: parseEventSpeakerExternalUser(user),
+      };
+    }
+
     if (!team) {
-      throw new Error('Team is required in event speaker');
+      throw new Error('Either team or external author is required');
```
Comment (markcsims) [inline apps/crn-server/src/entities/event.ts:96]: Should this be throwing GenericError?

File: apps/crn-server/test/entities/events.test.ts [hunk 2]
```diff
@@ -1,14 +1,21 @@
+import {
```
Comment (markcsims) [inline apps/crn-server/test/entities/events.test.ts:1]: Filename doesn't match the production name

File: packages/react-components/src/organisms/__tests__/SpeakerList.test.tsx [hunk 3]
```diff
@@ -66,6 +73,19 @@ describe('When rendering the speaker list', () => {
...
       expect(getByText('Genetics')).toBeVisible();
     });
+
+    it('Renders external author', async () => {
+      const event = {
+        ...createEventResponse(),
+        speakers: [externalSpeaker],
+      };
+
+      const { getByText } = render(<SpeakersList {...event} />);
```
Comment (markcsims) [inline packages/react-components/src/organisms/__tests__/SpeakerList.test.tsx:83]: prefer screen over destructuring the return of the render.

**Review by markcsims — CHANGES_REQUESTED** [review]

File: apps/crn-server/src/entities/event.ts (hunk 1 above)
Comment (niezgoda) [reply apps/crn-server/src/entities/event.ts:96]: GenericError is being thrown by APIs

**Review by markcsims — APPROVED** [review]

**Pull request merged**